build-livecd-root

```diff
@@ -23,6 +23,22 @@ fi
 
 which livemedia-creator ksflatten wget >/dev/null || ( echo "Command(s) missing, install required tools" && exit 2 )
 
+# Use custom setings from config file
+if [ -z ${DISTRO_INSTALL_ISO_URL} ]; then
+	DISTRO_INSTALL_ISO_URL="http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/images/boot.iso"
+fi
+
+if [ -z ${DISTRO_INSTALL_NAME} ]; then
+	DISTRO_INSTALL_NAME="Centos 8 Stream"
+fi
+
+if [ -n ${CUSTOM_LORAX_DIR} ]; then
+	LORAX_TEMPLATES="--lorax-templates ${CUSTOM_LORAX_DIR}"
+else
+	LORAX_TEMPLATES=""
+fi
```

Comment:
This is not correct shell code:

```
[lzap@nuc foreman-discovery-image]$ echo $CUSTOM_LORAX_DIR

[lzap@nuc foreman-discovery-image]$ [ -n ${CUSTOM_LORAX_DIR} ] && echo BAD
BAD
```

Your patch does not work when the variable is not set, then it breaks lorax as it passes the argument without any directory.